docs/design/access.md

```diff
@@ -83,6 +83,35 @@ K8s Cluster assets:
 
 This document is primarily about protecting K8s User assets and K8s cluster assets from other K8s Users and K8s Project and Cluster Admins.
 
+<!-- BEGIN MUNGE: EXAMPLE examples/guestbook/frontend-controller.yaml -->
+
+```yaml
+apiVersion: v1
+kind: ReplicationController
+metadata:
+  name: frontend
+  labels:
+    name: frontend
+spec:
+  replicas: 3
+  selector:
+    name: frontend
+  template:
+    metadata:
+      labels:
+        name: frontend
+    spec:
+      containers:
+      - name: php-redis
+        image: kubernetes/example-guestbook-php-redis:v2
+        ports:
+        - containerPort: 80
+
+```
+
+[Download example](http://releases.k8s.io/HEAD/examples/guestbook/frontend-controller.yaml)
```

Comment:
We should think about this link; it has to work
* from in github - which implies it should be relative, and not go through k8s.io
* from gh_pages, which implies it should be absolute. This is already a known bug, so I think we need to fix this when we make the gh_pages output.